src/poetry/puzzle/provider.py

```diff
@@ -49,6 +49,18 @@
 
 
 class Indicator(ProgressIndicator):
+    CONTEXT: str | None = None
+
+    @staticmethod
+    def set_context(context: str | None) -> None:
+        Indicator.CONTEXT = context
+
+    def _formatter_context(self) -> str:
+        if Indicator.CONTEXT is None:
+            return " "
+        else:
+            return f": {Indicator.CONTEXT} "
```

Comment:
```suggestion
            return f" <c1>{Indicator.CONTEXT}</> "
```

Reply:
@abn Shouldn't that be `</c1>` at the end?

Reply:
Both work from testing. But more specific is better I guess.